lib/fluent/plugin/out_sumologic.rb

```diff
@@ -293,6 +293,19 @@ def sumo_timestamp(time)
     time.to_s.length == 13 ? time : time * 1000
   end
 
+  # Convert log to string and strip it
+  def log_to_str(log)
+    if log.is_a?(Array) or log.is_a?(Hash)
+      log = Yajl.dump(log)
+    end
+
+    unless log.nil?
```

Comment:
Would it make sense to check if `log` responds to `strip!`?

Reply:
I don't feel that wrapping exception into another will change anything